.gitignore

```diff
@@ -43,3 +43,5 @@ __pycache__
 # Ignore gradle wrapper
 gradle/wrapper/gradle-wrapper.jar
 .gradletasknamecache
+
+.tool-versions
```

Comment:
anything to do with the fix? is this intended?

Reply:
Oh you're right, this is my asdf environment. Will remove it.